### PR TITLE
[MDB IGNORE] beer and ale headsmash fix 

### PR DESCRIPTION
**Pull request by manofpepsi**
## About The Pull Request

repaths the beer and ale to allow bottle headsmashing with them.

fixes:#60594

## Changelog
:cl: Pepsiman0
fix: you can now smash a bottle of beer or ale on someone's head
/:cl:

### CODE REVIEW

**Comment by TemporalOroboros** [comment]
Just a heads-up, but there's some office politics/drama going down right now. If someone starts a flame war in the comments feel free to ignore them. Things will calm down eventually.

**Comment by LemonInTheDark** [comment]
Fixes for the fix lord

**Comment by Fikou** [comment]
are you sure this wasnt intended? these bottles are smaller both in units and in sprite, they are cheaper in vendors and a lot easier to get than most headsmash thingies

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 1]
```diff
@@ -151,6 +151,31 @@
 	. = ..()
 	AddComponent(/datum/component/butchering, 200, 55)
 
+/obj/item/reagent_containers/food/drinks/bottle/beer
+	name = "space beer"
+	desc = "Beer. In space."
+	icon_state = "beer"
+	list_reagents = list(/datum/reagent/consumable/ethanol/beer = 30)
```
Comment (Ghommie) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm:158]:
```suggestion
	volume = 50
	list_reagents = list(/datum/reagent/consumable/ethanol/beer = 30)
```

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 2]
```diff
@@ -151,6 +151,31 @@
...
+	name = "Carp Lite"
+	desc = "Brewed with \"Pure Ice Asteroid Spring Water\"."
+	list_reagents = list(/datum/reagent/consumable/ethanol/beer/light = 30)
+
+/obj/item/reagent_containers/food/drinks/bottle/ale
+	name = "Magm-Ale"
+	desc = "A true dorf's drink of choice."
+	icon_state = "alebottle"
+	inhand_icon_state = "beer"
+	list_reagents = list(/datum/reagent/consumable/ethanol/ale = 30)
```
Comment (Ghommie) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm:175]:
```suggestion
	volume = 50
	list_reagents = list(/datum/reagent/consumable/ethanol/ale = 30)
```

**Review by Ghommie — COMMENTED** [review]
You forgot about their original volume.

**Comment by manofpepsi** [comment]
> are you sure this wasnt intended? these bottles are smaller both in units and in sprite, they are cheaper in vendors and you can get 10 free ones from bartenders closet

well I do presume that the bottle smashing with beer and ale bottles was intended just not realized for some reason, some stuff like the inhand icon states for the /obj/item/reagent_containers/food/drinks/bottle have beer in their names, and it would be just inconsistent with the rest of the bottled drinks that look similarly to beer but you can use the to bottle smash someone. also the small glass bottle which can be currently used for bottle smashing costs less than the beer in the booze-o-mat.

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm (hunk 1 above)
Comment (manofpepsi) [reply code/modules/food_and_drinks/drinks/drinks/bottle.dm:158]: thanks for pointing that out

**Review by Maurukas — APPROVED** [review]

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Pull request merged**